packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawerItem.tsx

```diff
@@ -147,7 +149,17 @@ export const NavigationDrawerItem = ({
       return;
     }
 
-    if (isNonEmptyString(to)) navigate(to);
+    if (isNonEmptyString(to)) {
+      const objectAndViewId = getObjectAndViewIdFromPath(to);
+      if (!isNull(objectAndViewId)) {
```

Comment:
!isDefined

Reply:
Won't require anymore, same reason as we changed to have in QueryParamsViewIdEffect.